data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1625 @@
+			label ssilvidacontinue
+			choice
+				`	(Agree)`
+				`	(Decline)`
+					decline
+			`	All four of them look pleased, and exchange looks that seem to indicate they expected that response. Taely gestures with a thankful tone. "We appreciate your willingness to help us explore this mystery, <first>. Given that time is of the essence here, this seems like an opportune time to reveal a new ship that has just reached production." She stands up, and gestures for you to accompany her to the shipyard.`
+			`	There, crouched next to <ship>, is what you can only describe as a predator: It has long sleek lines with flared wings that resemble feathers, and a massive engine array that suggest incomparable mobility in a ship of its size. As you approach you can see several Remnant working around it. One gestures, and the ship extends a wing and the Remnant reaches in between the feathers to do something, then steps back and the ship runs through a range of twisting motions as it moves its wings around. It strongly resembles a raptor preening its feathers.`
+			`	Taely walks right up to the nose and the ship crouches down so she can put her hand on its nose. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful."`
+			choice
+				`	"Looks great. Let's fly!"`
+					goto letsfly
+				`	"What sort of ship is it?"`
+			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are conscious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`
```

Comment:
This raises some... interesting humanitarian thoughts. Do the ships get to _choose_ whether they're put in harm's way? Reminds me a little of using, say, horses in a medieval war. I don't think this is by any means something I'm concerned about, but I do think there's room for some interesting discussion about this. Maybe for future content? Something to consider incorporating into future Remnant stories if you weren't already.

Reply:
Yep, it's already being planned. Nice to see you pick up on it. :)